bakery/src/tasks/upload-docx.js

```diff
@@ -0,0 +1,53 @@
+const dedent = require('dedent')
+
+const { constructImageSource } = require('../task-util/task-util')
+
+const task = (taskArgs) => {
+  const { parentGoogleFolderId, awsAccessKeyId, awsSecretAccessKey, queueStateBucket, codeVersion } = taskArgs
+  const imageDefault = {
+    name: 'openstax/cops-bakery-scripts',
+    tag: 'trunk'
+  }
+  const imageOverrides = taskArgs != null && taskArgs.image != null ? taskArgs.image : {}
+  const imageSource = constructImageSource({ ...imageDefault, ...imageOverrides })
+
+  return {
+    task: 'upload docx',
+    config: {
+      platform: 'linux',
+      image_resource: {
+        type: 'docker-image',
+        source: imageSource
+      },
+      params: {
+        GOOGLE_SERVICE_ACCOUNT_CREDENTIALS: '((google-service-account-credentials))',
+        AWS_ACCESS_KEY_ID: `${awsAccessKeyId}`,
+        AWS_SECRET_ACCESS_KEY: `${awsSecretAccessKey}`
+      },
+      inputs: [
+        { name: 'book' },
+        { name: 'docx-book' }
+      ],
+      run: {
+        path: '/bin/bash',
+        args: [
+          '-ce', // FIXME: Not including -x to avoid displaying credentials when writing to file.
```

Comment:
could you `set -x` after the credentials line?

Reply:
Oh, man -- of course. Thanks!